mpqp/core/instruction/gates/native_gates.py

```diff
@@ -1117,6 +1249,12 @@ def qiskit_gate(cls):
 
         return PhaseGate
 
+    @classproperty
+    def cirq_gate(cls):
+        from cirq.ops.common_gates import ZPowGate
+
+        return lambda theta: ZPowGate(exponent=theta / np.pi)
```

Comment:
How do you make the link between `k` and `theta` here ?

Reply:
it's a lamba function we pass theta in the RotationGate class to_otherLanguages()